kong/kong.lua

```diff
@@ -144,9 +144,11 @@ function Kong.init()
 end
 
 function Kong.init_worker()
+  prepare_module_variables()
```

Comment:
Globals initialized in the `init` phase should be present in `init_worker` too, or did you run in an edge-case?